pylutron_caseta/pairing.py

```diff
@@ -187,8 +188,8 @@ async def _async_verify_certificate(server_addr, signed_ssl_context):
 
 
 def _generate_csr_with_ssl_context():
-    with tempfile.NamedTemporaryFile() as lap_cert_temp_file:
-        with tempfile.NamedTemporaryFile() as lap_key_temp_file:
+    with tempfile.NamedTemporaryFile(delete=not IS_WINDOWS) as lap_cert_temp_file:
```

Comment:
The Windows code works for all operating systems. It's easier to only write one version of the code.